ios/MullvadVPN/View controllers/Preferences/PreferencesDataSource.swift

```diff
@@ -249,14 +267,22 @@ final class PreferencesDataSource: UITableViewDiffableDataSource<
     }
 
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return UIMetrics.sectionSpacing
+        let sectionIdentifier = snapshot().sectionIdentifiers[section]
+
+        switch sectionIdentifier {
+        case .contentBlockers:
+            return UITableView.automaticDimension
```

Comment:
I think maybe it's better to set fixed value for header. automatic dimension caused overhead while the header is not going to be grown up dynamically